resharper/resharper-unity/src/CSharp/Daemon/Stages/PerformanceCriticalCodeAnalysis/Highlightings/PerformanceHighlightingAttributeIds.cs

```diff
@@ -1,6 +1,57 @@
+using JetBrains.ReSharper.Plugins.Unity.CSharp.Daemon.Stages.Highlightings;
+using JetBrains.TextControl.DocumentMarkup;
+
 namespace JetBrains.ReSharper.Plugins.Unity.CSharp.Daemon.Stages.PerformanceCriticalCodeAnalysis.Highlightings
 {
-    // Note that all attribute IDs should start with "ReSharper Unity " to appear properly in both Rider and ReSharper
+    // Note that these are registered as part of the main "Unity" group, as they should appear in the main "Unity"
+    // Color Scheme page
+    [RegisterHighlighter(COSTLY_METHOD_INVOCATION,
+        GroupId = UnityHighlightingAttributeIds.GROUP_ID,
+        EffectType = EffectType.SOLID_UNDERLINE,
+        EffectColor = "#ff7526",
+        NotRecyclable = true,
+        Layer = HighlighterLayer.SYNTAX)]
```

Comment:
I'd also use ADDITIONAL_SYNTAX or even higher, and leave syntax to lexer-based highlighting (comments, literals, etc.)